contrib/datawave-quickstart/bin/services/accumulo/install.sh

```diff
@@ -59,9 +59,9 @@ fi
 assertCreateDir "${DW_ACCUMULO_JVM_HEAPDUMP_DIR}"
 
 # Update tserver and other options in accumulo-env.sh
-sed -i "s~\(ACCUMULO_TSERVER_OPTS=\).*$~\1\"${DW_ACCUMULO_TSERVER_OPTS}\"~g" ${DW_ACCUMULO_CONF_DIR}/accumulo-env.sh
-sed -i "s~\(export JAVA_HOME=\).*$~\1\"${JAVA_HOME}\"~g" ${DW_ACCUMULO_CONF_DIR}/accumulo-env.sh
-sed -i "s~\(export ACCUMULO_MONITOR_OPTS=\).*$~\1\"\${POLICY} -Xmx2g -Xms512m\"~g" ${DW_ACCUMULO_CONF_DIR}/accumulo-env.sh
+sed -i'.bak' "s~\(ACCUMULO_TSERVER_OPTS=\).*$~\1\"${DW_ACCUMULO_TSERVER_OPTS}\"~g" ${DW_ACCUMULO_CONF_DIR}/accumulo-env.sh && rm ${DW_ACCUMULO_CONF_DIR}/accumulo-env.sh.bak
+sed -i'.bak' "s~\(export JAVA_HOME=\).*$~\1\"${JAVA_HOME}\"~g" ${DW_ACCUMULO_CONF_DIR}/accumulo-env.sh && rm ${DW_ACCUMULO_CONF_DIR}/accumulo-env.sh.bak
+sed -i'.bak' "s~\(export ACCUMULO_MONITOR_OPTS=\).*$~\1\"\${POLICY} -Xmx2g -Xms512m\"~g" ${DW_ACCUMULO_CONF_DIR}/accumulo-env.sh && rm ${DW_ACCUMULO_CONF_DIR}/accumulo-env.sh.bak
```

Comment:
Why not just `sed -i ''` instead of `sed -i '.bak'` and then you don't have to remove the `.bak` file separately?